packages/devextreme/scss/widgets/material/tabs/layout/tab/styling-mode/_secondary.scss

```diff
@@ -15,8 +15,12 @@
 
 
 .dx-tabs-styling-mode-secondary {
-  .dx-tab .dx-tab-text {
-    text-transform: initial;
+  .dx-tab {
+    min-width: 0;
+
+    .dx-tab-text {
+      text-transform: initial;
```

Comment:
The initial value of `text-transform` property is 'none'. Why we don't you it here?

Reply:
[40e6acc34096cc1c7a90be14077310520364cd6d]